src/pybind/mgr/dashboard/frontend/src/app/shared/models/cephfs.model.ts

```diff
@@ -3,3 +3,20 @@ export interface MountData {
   fsName: string;
   path: string;
 }
+
+export interface CephCaps {
+  mds?: string;
+  mgr?: string;
+  mon?: string;
+  osd?: string;
+}
+
+export interface CephUser {
+  entity: string;
+  caps: CephCaps;
+  key: string;
+}
+
+export interface MonitorData {
+  in_quorum: { public_addr: string }[];
+}
```

Comment:
and before i forget, these are not ceph-fs model. check if we have a model for mapping ceph client users and if not let's create a new model for them.